### ISSUE TITLE
Add time-to-accuracy speedup plot

### ISSUE THREAD

**Issue by martinjaggi** [mlbench/mlbench-benchmarks#7]
in the benchmark description and results page, add a time-to-accuracy speedup plot, next to the throughput plot. this measures relative speedup in time compared to N=1 nodes.

it is crucial to point out how the same accuracy is (relatively) much slower to reach as the number of machines grows, i.e. the issues of large-batch training.

this is relevant for both cifar10 and linear models currently.

**Comment by martinjaggi** [comment]
is there an update on this one?

**Comment by negar-foroutan** [comment]
I added the plots for linear model a month ago.

**Comment by tlin-taolin** [comment]
I've discussed with Lie and checked the existing results in this page
https://mlbench.readthedocs.io/en/develop/benchmark-tasks.html

For Deep Learning results (what Lie did), we have ResNet on CIFAR-10.
However, (1) the speedup of time-to-accuracy is compared to N=2 nodes; (2)
the learning rate is not fine-tuned for each case; instead, the figure just
compares two cases (w/ and w/o learning rate scaling rule); (3) the
description of the Figure might need to be improved and should be more
precise (e.g., it is not very clear to me which ResNet architecture we are
using); (3) two metrics for comparison are mentioned, i.e.,
Accuracy-after-Time and Time-to-Accuracy, but only the results of
Time-to-Accuracy are presented.

Regarding the experiments that I did, I found it is a bit hard to directly
merge them to the current results, as (1) we are using a different platform
for the time evaluation, (2) we are using a slightly hyper-parameter for
training (I trained a bit longer and the local mini-batch size differed a
bit). So @Martin, please let me know how can I contribute (it is still a
bit unclear to me w.r.t. your expected benchmarking results). Actually, in
terms of the benchmarking ResNet on CIFAR-10, the current results are
somehow complete but we might need to slightly adjust the figures a bit for
better presentation.

**Comment by martinjaggi** [comment]
ok. 1) we need the speedup relative to N=1 nodes
2) we only need one regime of learning rate schedule (as simple as possible), but one we're sure it scales well with all the N we have. we should also add a footnote how we selected that regime and how much worst it is for the highest N compared to a tuned learning rate
3) ok

if you can help adjusting it to the precise benchmark task (which we can also adjust), would be great. as you prefer

**Comment by martinjaggi** [comment]
also the first resnet plot on the result page can be deleted, as well as throughput.

cost we'll keep, but just as number (estimate to run the whole benchmark) is ok, no 'cost scaling' plot necessary for now
